packages/connect-node/src/grpc-web-transport.ts

```diff
@@ -166,8 +167,7 @@ export function createGrpcWebTransport(
             body: pipe(
               createAsyncIterable([req.message]),
               transformSerializeEnvelope(
-                serialization.getI(opt.useBinaryFormat),
-                opt.writeMaxBytes
```

Comment:
The I/O limit is now added to `Serialization` via `limitSerialization()`. That means we can apply limits to Connect unary without going through boilerplate.